jobsdb/migration.go

```diff
@@ -459,18 +459,85 @@ func (jd *Handle) getMigrationList(dsList []dataSetT) (migrateFrom []dsWithPendi
 	return
 }
 
+func getColumnConversion(srcType, destType string) string {
```

Comment:
Is it possible to find a universal way to do the mapping, that will always work?